build.gradle

```
@@ -154,7 +159,7 @@ task uberJar(type: Jar) {
             'Build-OS'       : "${System.properties['os.name']} ${System.properties['os.arch']} ${System.properties['os.version']}"
         )
     }
-    baseName = 'synthea-with-dependencies'
+    archiveBaseName = 'synthea-with-dependencies'
     from { configurations.compile.collect { it.isDirectory() ? it : zipTree(it) } }
```

Update to the previous comment, this seems to work based on https://stackoverflow.com/a/4894308 and a quick local test

```suggestion
    from { configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) } }
```

---

I confirmed the issue and your suggested fix. Thanks!